terraform/environments/xhibit-portal/network-infrastructure.tf

```diff
@@ -448,6 +499,27 @@ resource "aws_security_group_rule" "portal-http-to-waf-lb" {
   source_security_group_id = aws_security_group.waf_lb.id
 }
 
+resource "aws_security_group_rule" "portal-http-from-prtg-lb" {
+  depends_on               = [aws_security_group.prtg_lb, aws_security_group.portal_server]
+  security_group_id        = aws_security_group.portal_server.id
+  type                     = "ingress"
+  description              = "allow access to prtg Load Balancer from portal server"
```

Comment:
Not needed - we're not accessing the portal servers from the PRTG load balancer

Reply:
Will test this change

Reply:
updated.. remarked resources portal-http-from-prtg-lb